pages/Frequency/Publishing.md

```diff
@@ -1,14 +1,13 @@
 # Announcement Publishing
 
 On Frequency, [Announcements](../DSNP/Announcements.md) are mapped to Schemas which in turn publish Frequency Messages.
-Frequency Messages are either direct Graph Changes from a particular user, or a Batch Publication with a multitude of possible users.
+Frequency Messages are either mutable state changes from a particular user, or a Batch Publication with a multitude of possible users.
```

Comment:
Messages are never mutable state changes, but I get what is trying to happen here.

Perhaps:

```suggestion
On Frequency, [Announcements](../DSNP/Announcements.md) are mapped to Schemas which in turn publish Frequency Messages or state changes.
Frequency Messages can be from a particular user, or a Batch Publication with a multitude of possible users.
```